packages/discovery-react-components/src/components/StructuredQuery/__tests__/StructuredQuery.test.tsx

```diff
@@ -29,12 +89,6 @@ describe('<StructuredQuery />', () => {
           expect(addRuleRowText).toBeDefined();
           expect(addGroupRulesText).toBeDefined();
         });
-        test('field dropdown has the correct default messages', () => {
```

Comment:
why is this being removed?

Reply:
This test was placed under the `when there are no fields present` block